.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: go
 
 go:
-  - 1.7
+  - 1.7.x
```

Comment:
I've seen tons of repos use `1.7` or `1.7.4`, so definitely be clear about this and recommend it in the docs.